cspm/rn/prisma-cloud-compute-release-information/features-introduced-in-compute-april-2023.adoc

```diff
@@ -192,7 +192,7 @@ Every https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-compute
 [#id-backward-compatibility]
 === Backward Compatibility for New Features
 
-NOTE: With this release, Defenders running versions 22.01 or earlier will no longer be able to connect to the console. 
+NOTE: With this release, Defenders running versions earlier than 30.00 will no longer be able to connect to the console. 
```

Comment:
@divyabhushan why did we add this? Was this a PM suggestion?

Reply:
Yes, this can be removed. This was a similar note as added in 22.12.427 RN. Thank you